tests/test_templates.py

```diff
@@ -12,106 +12,307 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Unit tests for the integration of the methods from :mod:`pennylane.init` with the :mod:`pennylane.templates` module.
+Integration tests for templates, including integration of passing outputs of initialization functions
+in :mod:`pennylane.init`, and running templates in larger circuits.
 """
 # pylint: disable=protected-access,cell-var-from-loop
 import pytest
+import torch
```

Comment:
Rather than importing directly, we should make use of the `torch_support` and `tf_support` fixtures

Reply:
(you can have a look at `test_vqe.py` for an example of a test files that tests all three interfaces)

Reply:
To work with the parametrized interfaces, I now add interfaces to the tests only if the ``try: import`` statement succeeds.